app/src/main/java/org/wikipedia/main/MainActivity.java

```diff
@@ -207,4 +219,16 @@ protected void setToolbarElevationDefault() {
     protected void clearToolbarElevation() {
         getToolbar().setElevation(0f);
     }
+
+    private void dismissCurrentTooltip() {
+        if (currentTooltip != null) {
+            currentTooltip.dismiss();
+        }
+        currentTooltip = null;
+    }
+
+    public void setCurrentTooltip(Balloon tooltip) {
```

Comment:
Can we have a `@NonNull` for the parameter?

Reply:
ok!